### PR TITLE
Add support for Jupiter LP token

### PR DESCRIPTION
**Pull request by oeble**

### CODE REVIEW

File: programs/scope/tests/tests_security_refresh_list.rs [hunk 1]
```diff
@@ -1187,3 +1197,51 @@ mod ktoken_tests {
         );
     }
 }
+
+// - [x] Wrong Jupiter LP additional mint account
+#[tokio::test]
+async fn test_refresh_one_jlp_wrong_mint() {
```
Comment (elliotkennedy) [inline programs/scope/tests/tests_security_refresh_list.rs:1203]:
```suggestion
async fn test_refresh_list_jlp_wrong_mint() {
```

**Review by elliotkennedy — APPROVED** [review]
Great work!

**Review by silviutroscot — APPROVED** [review]

File: configs/mainnet/hubble.json [hunk 2]
```diff
@@ -565,5 +565,11 @@
     "oracle_mapping": "2fnxDPcLAZzYy1pJTMFNUo2mxWHD4ggrEgTu6my54Npm",
     "oracle_type": "SwitchboardV2",
     "max_age": 750
+  },
+  "124": {
+    "label": "JLP/USD",
+    "oracle_mapping": "5BUwFW4nRbftYTDMbgxykoFWqWHPzahFSNAaaaJtVKsq",
+    "oracle_type": "JupiterLP",
+    "max_age": 100
```
Comment (alittlezz) [inline configs/mainnet/hubble.json:573]: Isn't this too low? Isn't this pool price modified only when someone interacts with the protocol?

File: programs/scope/src/oracles/jupiter_lp.rs [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        // TODO: find a way to get the last update time
+        last_updated_slot: clock.slot,
+        unix_timestamp: u64::try_from(clock.unix_timestamp).unwrap(),
+        ..Default::default()
+    };
+
+    Ok(dated_price)
+}
+
+pub fn validate_jlp_pool(account: &AccountInfo) -> Result<()> {
```
[the remark this answers is not in the document]
Comment (oeble) [reply programs/scope/src/oracles/jupiter_lp.rs:54]: This is used when updating the mapping. It is a sanity check that the admin is not doing sh**.

File: configs/mainnet/hubble.json (hunk 2 above)
Comment (oeble) [reply configs/mainnet/hubble.json:573]: We don't have any time information on the last refresh time of the account so I prefer faster than slower.

**Pull request merged**